baselines/constraint/constraint_wrapper.py

```diff
@@ -41,6 +41,27 @@ def __init__(self,
             self.logs = None
 
     def augment_obs(self, ob):
+        """Augment the current observation.
+
+        Parameters
+        ----------
+        self: ConstraintEnv
+            A data structure that serves as 
+            a wrapper to the Gym environment.
+
+        ob: Tensor?
+            A Tensor? or data structure that
+            represents the observation of the
+            state.
+        
+        Returns
+        -------
+        ob: Tensor?
```

Comment:
It's going to depend on what setting you're using, but generally it's going to be an iterable which contains the observation and the state of the constraints